src/Illuminate/Queue/Worker.php

```diff
@@ -218,8 +241,6 @@ public function runNextJob($connectionName, $queue, WorkerOptions $options)
         if ($job) {
             return $this->runJob($job, $connectionName, $options);
         }
-
-        $this->sleep($options->sleep);
```

Comment:
Tried to find a nice solution to force the sleep time to 0 if 'once' is supplied, but given the current flow (queue:listen calling queue:work as an external command) there is no lean way. So I just brought it back. Unless user supplies --sleep=0, queue:work --once will still sleep for default 3 seconds (which makes no sense, but we can wait till 5.5 I suppose?)